catalog/USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER

```diff
@@ -0,0 +1,24 @@
+'gee:skip_indexing': true,
```

Comment:
'gee:skip_indexing': true should be a part of JSON - compare with https://github.com/google/earthengine-catalog/blob/main/catalog/GOOGLE/GOOGLE_GLOBAL_CCDC_V1.jsonnet#L15

Sorry, I added the comment to line 1 just to make it a general file comment.